File: Models/Company.cs
using System;
using System.Collections.Generic;

namespace BackboneData.Models
{
    public class Company
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public string Industry { get; set; }
        public string Size { get; set; }
        public string TargetMarket { get; set; }
        public string OperatingRegion { get; set; }
        public string Email { get; set; }
        public string Fax { get; set; }
        public string WebsiteLink { get; set; }
        public string PhoneNumber { get; set; }
        public string Agent { get; set; }
        public string LandLine { get; set; }
        public string UpdatedAt { get; set; }
        public string Remarks { get; set; }
        public List<Stage> Stages { get; set; }
    }
}

File: client-app/src/models/company.ts
import { Stage } from "./stage";

export interface Company {
    id: string;
    name?: string;
    location?: string;
    industry?: string;
    agent?: string;
    size?: string;
    targetMarket?: string;
    operatingRegion?: string;
    email?: string;
    fax?: string;
    websiteLink?: string;
    phoneNumber?: string;
    landLine?: string;
    updatedAt?: string;
    remarks?: string;
    stages?: Stage[]
}